howto.md

```diff
@@ -0,0 +1,109 @@
+
+Part two: connect the Pebble app to the Rebble Web Services.
+
+* On the phone, open the default browser (most likely Chrome), and go to [auth.rebble.io](https://auth.rebble.io/).
+* You are greeted with a number of ways to sign in. Choose one and sign in. **Remember your sign in credentials!** You will return to the Rebble site, and are greeted with "You're all set!". Read the page, then tap the "get started now" link in the first paragraph.
+* You are redirected to [boot.rebble.io](https://boot.rebble.io/), and are greeted with "Here we go!". Read the page, then tap the "switch to rebble" button. The browser asks if you want to open it with Pebble. Tap "Open".
+* The Pebble app opens. One of three things will then happen:
+* * A Rebble screen will appear. It may have an orange spinner above it; you may ignore this.
+* * The old Pebble login screen will appear. If this happens, close the app from the app switcher and open it again. A Rebble screen should now appear.
+* * Nothing. If this happens, tap ⋮ in the top right corner, then tap Settings. At this point, either:
+* * * A Rebble screen will appear, or
+* * * Nothing interesting. In this case, scroll down to "ACCOUNT" and make sure you are logged out. If not, tap it to log out. Now, tap to log in. A Rebble screen should appear.
+* In any case, once you tap the button on the Pebble app's "Welcome to Rebble" screen, you are done!
+* If your Pebble disconnects, just tap the usual connect button.
+* You're done! Go to "Watchfaces", tap the big red "+" in the lower right corner and find and install watchfaces you like from the Rebble store.
+
+### Weather and dication services
+
+So, you started using the Rebble Web Services, but want to add the paid plan for weather and dictation. There are a few things to consider, before you order it:
+* On iOS, replying to text messages isn't possible. This is a limitation from iOS. So, dictation will NOT add spoken replies to text messages.
+* Weather services is solely for the native Pebble weather app and for weather info in timeline. Watchfaces and 3rd party apps that offer weather info, have their own means to fetch weather info. You don't need the Rebble paid plan for them.
+* Payment is handled by Stripe, which only offers payment by credit card.
+
+Still want it? Let's go!
+* Open the default browser on your phone and head to [Rebble Account](https://auth.rebble.io/account/).
+* **Very important: make sue you are signed in with the same account you used when you first made the switch to Rebble!**
+* Choose your plan ($3.00/month or $33/year).
+* You are forwarded to Stripe. Enter your card info and finish the payment.
+* You are redirected to the Rebble site. Voice / Weather subscription should now be **Active**.
+* In the last paragraph, tap the "repeat the initial setup" link. Then, follow the rest of the procedure for stap 2 above.
```

Comment:
what do you think about a "thanks for supporting Rebble" bullet point as well or something? :bowtie:

Reply:
Done!